server/helpers/adapters_tracker.go

```diff
@@ -119,11 +125,99 @@ func (a *AdaptersTracker) DeployAdapter(ctx context.Context, adapter models.Adap
 			return ErrAdapterAdministration(err)
 		}
 
+	case "kubernetes":
+		kubeClient, err := meshkitkube.New([]byte(""))
+		if err != nil {
+			return ErrAdapterAdministration(err)
+		}
+
+		exposedPort, _ := strconv.Atoi(adapter.Location)
+		adapterPort := 0
+		for _, availableAdapter := range models.ListAvailableAdapters {
+			if availableAdapter.Name == adapter.Name {
+				adapterPort, _ = strconv.Atoi(availableAdapter.Location)
+				break
+			}
+		}
+
+		// Create a deployment
+		deployment := &appsv1.Deployment{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      adapter.Name,
+				Namespace: core.MesheryNamespace,
+			},
+			Spec: appsv1.DeploymentSpec{
+				Selector: &metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"app": adapter.Name,
+					},
+				},
+				Replicas: int32Ptr(1),
+				Template: corev1.PodTemplateSpec{
+					ObjectMeta: metav1.ObjectMeta{
+						Labels: map[string]string{
+							"app": adapter.Name,
+						},
+					},
+					Spec: corev1.PodSpec{
+						Containers: []corev1.Container{
+							{
+								Name:  adapter.Name,
+								Image: "layer5/" + adapter.Name + ":stable-latest",
```

Comment:
Ideally, we don't presume the container registry org name. Let's externalize this configuration, if we can.

Reply:
Doing so might require a follow up PR.

Reply:
Externalise as in? take through user input?